config_test.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+import (
+	"testing"
+)
+
+func TestClone(t *testing.T) {
+	c := new(Config)
+	c.Identity.PeerID = "faketest"
+	c.API.HTTPHeaders = map[string][]string{"foo": []string{"bar"}}
```

Comment:
To be complete I would also add a test for modifying an existing value of a map, rather then creating a new one.